### PR TITLE
Refactor FXIOS-7256 [v119] LibraryCoordinator nimbus feature to true

### PR DESCRIPTION
**Pull request by FilippoZazzeroni**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7256)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/16088)

## :bulb: Description
Enable `LibraryCoordinator` refactor by default to true.

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [x] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [x] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Review by yoanarios — APPROVED** [review]

File: nimbus-features/libraryCoordinatorRefactor.yaml [hunk 1]
```diff
@@ -15,5 +15,5 @@ features:
           enabled: false
       - channel: developer
         value:
-          enabled: false
```
Comment (lmarceau) [inline nimbus-features/libraryCoordinatorRefactor.yaml:18]: We should turn ON in Beta and by default too 🙏 same for the other PR

**Comment by yoanarios** [comment]
@FilippoZazzeroni there is a couple of unit test that are failing could you take a look please

File: nimbus-features/libraryCoordinatorRefactor.yaml (hunk 1 above)
Comment (FilippoZazzeroni) [reply nimbus-features/libraryCoordinatorRefactor.yaml:18]: Ok i'm doining it now

**Comment by yoanarios** [comment]
> @yoanarios The tests are now fixed with the last commit

Thank you, I am running the build again

**Pull request merged**